web_widget_float_formula/static/tests/js/test_web_widget_float_formula.js

```diff
@@ -116,10 +117,10 @@ odoo.define_section('web_widget_float_formula', ['web.form_common', 'web.form_wi
     test('._compute_result() should behave properly when the current value of the input element is a valid formula',
         function(assert, form_common, form_widgets, core) {
             window.test_setup(this, form_common, form_widgets, core);
-            this.$element.val('=2*3');
+            this.field.$el.val('=2*3');
```

Comment:
`this.$element` and `this.field.$el` are functionally equivalent, but the former is more explicit. 

The reasoning behind this is that `this.field.$el` is a jQuery collection (array basically) that contains `this.$element`. jQuery operates on collections as a package, so `.val` is just setting all elements in the collection (`[this.$element]`) to the value.